dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -4,11 +4,18 @@ class Pd::ProfessionalLearningLandingController < ApplicationController
   def index
     # Get the courses that this user teaches
     workshops = Pd::Workshop.enrolled_in_by(current_user)
```

Comment:
Yeah, I'm fine with this. If you've enrolled in a CSF class, you're a CSF teacher.